common/tmx_filetype.go

```diff
@@ -0,0 +1,62 @@
+package common
+
+import (
+	"bytes"
+	"engo.io/engo"
```

Comment:
Since I've already made enough comments to fill another commit: every package not within the standard library goes within its own group. 

So:

```go
import (
     "bytes"
     "fmt"
    "io"

    "engo.io/engo"
)
```

The tool `goimports` might help with that (that one splits the two groups, as is recommended by the Go team)). That basically does a `gofmt` with an organization of the `import` statement afterwards.